modules/mongodb/templates/mongodb-restore-s3.erb

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -e
+
+# Redirect stdout and stderr to syslog
+exec 1> >(/usr/bin/logger -s -t $(basename $0)) 2>&1
+
+S3_BUCKET=<%= @s3_bucket %>
+BACKUP_DIR=<%= @backup_dir %>
+PRIMARY=$(mongo --quiet --eval 'db.isMaster().ismaster')
+
+cd $BACKUP_DIR
+
+
+TIME="$(date +%s)"
+
+# Download and decrypt backup
+/usr/bin/envdir <%= @env_dir %>/env.d /usr/local/bin/s3cmd get --force `/usr/local/bin/s3cmd ls s3://${S3_BUCKET}/mongodump* | \
+tail -1 | awk '{print $4}'`| awk '{print $4}' | tr -d "'" | \
+xargs /usr/bin/gpg --yes --quiet --output mongodump.tar.gz --decrypt && tar xzf mongodump.tar.gz
```

Comment:
This restore job will be pulling Production data to Staging and Integration machines, so they'll have different keys. We might need the Production keys on these machines, which isn't great, but it's better to be badly encrypted than not encrypted at all.